### PR TITLE
feat(option): adds "canUpdateSelector" option

### PR DESCRIPTION
**Pull request by stalniy**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -100,7 +104,7 @@ function Angular2ConventionsLoader(source, sourcemap) {
           return 'template:' + replaceStringsWithRequires(url);
         })
     }
-    if (decorator === 'Component' && templateUrlRegex.test(metadata)) {
+    if (decorator === 'Component' && styleUrlsRegex.test(metadata)) {
```
Comment (gdi2290) [inline index.js]: why did you change this?

Comment (stalniy) [inline index.js]: I thought this was done by mistake. Because here we need to check if there are style urls and if so, update them.

Comment (gdi2290) [inline index.js]: you're right :)
